src/libs/actions/PersonalDetails.js

```diff
@@ -227,6 +227,21 @@ function setAvatar(file) {
     });
 }
 
+/**
+ * Deletes the user's avatar image
+ *
+ * @param {String} login
+ */
+function deleteAvatar(login) {
+    // We don't want to save default avatar on BE, just update to empty string
```

Comment:
`on BE` might be a little unclear. I would say something like

```
We don't want to save the default avatar URL in the backend since we don't want to allow users the option of removing the default avatar, instead we'll save an empty string
```